Directory.Packages.props

```diff
@@ -27,7 +27,7 @@
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Timer" Version="4.2.0" />
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Storage.Queues" Version="5.1.2" />
     <PackageVersion Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
-    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="6.0.1" />
```

Comment:
Did you go through and check all the transitive dependencies as well?

Also - while you're here could you add a comment to the SqlClient package entry explaining that we can't bump beyond 5.0.1 right now and link to https://github.com/Azure/azure-functions-sql-extension/issues/920?